health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppFieldComposer.js

```
@@ -0,0 +1,600 @@
+                      ? "selected"
+                      : ""
+                  }`
+                : ""
+            }
+          >
+            <div className="appConfiglabelField-label">
+              <span>{`${t(label)}`}</span>
+              {Mandatory && <span className="mandatory-span">*</span>}
+              {helpText && (
+                <span className="icon-wrapper">
+                  <TooltipWrapper content={t(helpText)} children={<InfoOutline fill={"#C84C0E"} width={"20px"} height={"20px"} />} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)